vernac/comAssumption.ml

```diff
@@ -32,7 +32,7 @@ let declare_variable is_coe ~kind typ univs imps impl name =
   let () =
     if is_coe = Vernacexpr.AddCoercion then
       ComCoercion.try_add_new_coercion
-        r ~local:true ~reversible:true in
+        r ~local:true ~reversible:false in
```

Comment:
Should `reversible` be optional (defaulting to false)?

Reply:
Probably yes. I guess, my initial intent in not making optional was to avoid forgetting a callsite. But I managed to screw up anyway, so that was probably a bad choice.